gui/public/i18n/ru/translation.ftl

```diff
@@ -0,0 +1,542 @@
+onboarding-continue = Продолжить
+onboarding-wip = В работе W.I.P.
+
+## Wi-Fi setup
+onboarding-wifi_creds-back = Вернуться к введению
+onboarding-wifi_creds = Вставьте данные Wi-Fi
+# This cares about multilines
+onboarding-wifi_creds-description =
+    Трекеры будут использовать эти учетные данные для беспроводного подключения.
+    Пожалуйста, используйте данные Wi_Fi, к которому вы в данный момент подключены.
+onboarding-wifi_creds-skip = Пропустить настройки Wi-Fi
+onboarding-wifi_creds-submit = Отправить!
+onboarding-wifi_creds-ssid =
+    .label = Имя Wi-Fi
+    .placeholder = Вставьте имя Wi-Fi
+onboarding-wifi_creds-password =
+    .label = Пароль
+    .placeholder = Вставьте пароль Wi-Fi
+
+## Mounting setup
+onboarding-reset_tutorial-back = Вернитесь к калибровке крепления
+onboarding-reset_tutorial = Сбросить туториал
+onboarding-reset_tutorial-description = Эта функция не завершена, просто нажмите продолжить
+
+## Setup start
+onboarding-home = Добро пожаловать в SlimeVR!
+# This cares about multilines and it's centered!!
+onboarding-home-description =
+    Обеспечение полного отслеживания тела
+    для всех
+onboarding-home-start = Давайте все настроим!
+
+## Enter VR part of setup
+onboarding-enter_vr-back = Вернуться к привязке трекеров
+onboarding-enter_vr-title = Время зайти в VR!
+onboarding-enter_vr-description = Наденьте все ваши трекеры и зайдите в VR!
+onboarding-enter_vr-ready = Я готов
+
+## Setup done
+onboarding-done-title = Вы готовы!
+onboarding-done-description = Наслаждайтесь игре!
+onboarding-done-close = Закрыть гид
+
+## Tracker connection setup
+onboarding-connect_tracker-back = Вернитесь к данным Wi-Fi
+onboarding-connect_tracker-title = Подключить трекеры
+onboarding-connect_tracker-description-p0 = Теперь самое интересное - подключение всех трекеров!
+onboarding-connect_tracker-description-p1 = Просто подключите все, что еще не подключены, через USB-порт.
+onboarding-connect_tracker-issue-serial = У меня проблемы с подключением!
+onboarding-connect_tracker-usb = USB Трекер
+onboarding-connect_tracker-connection_status-connecting = Отправить данные Wi_Fi
+onboarding-connect_tracker-connection_status-connected = Подключен к Wi-Fi
+onboarding-connect_tracker-connection_status-error = Невозможно подключиться к Wi-Fi
+onboarding-connect_tracker-connection_status-start_connecting = Ищем трекеры...
+onboarding-connect_tracker-connection_status-handshake = Подключен к трекеру
+# $amount (Number) - Amount of trackers connected (this is a number, but you can use CLDR plural rules for your language)
+# More info on https://www.unicode.org/cldr/cldr-aux/charts/22/supplemental/language_plural_rules.html
+# English in this case only has 2 plural rules, which are "one" and "other",
+# we use 0 in an explicit way because there is no plural rule in english for 0, so we directly say
+# if $amount is 0 then we say "No trackers connected"
+onboarding-connect_tracker-connected_trackers = { $amount ->
```

Comment:
I think this isn't correct. The current code will give the following results:
| Tracker Number  | Text displayed |
| ------------- | ------------- |
| 0  | Нет трекеров  |
| 1  | 1 трекер  |
| 2  | 2 трекера |
| 3  | 3 трекера |
| 4  | 4 трекера |
| 5  | трекеров |
| 6..10  | трекеров |
| 11 | 1 трекер |

As you can see the number dissapears for 5-10, and 11 isn't shown as 11 but 1 instead.
I don't know russian myself but I think the following is more correct:
```
onboarding-connect_tracker-connected_trackers = { $amount ->
    [0] Нет трекеров
    [one] { $amount } трекер
    [few] { $amount } трекера
    *[other] { $amount } трекеров
} подключено
```
This will give the following table:
| Tracker Number  | Text displayed |
| ------------- | ------------- |
| 0  | Нет трекеров  |
| 1  | 1 трекер  |
| 2  | 2 трекера |
| 3  | 3 трекера |
| 4  | 4 трекера |
| 5  | 5 трекеров |
| 6  | 6 трекеров |
| 7  | 7 трекеров |
|  | (8,9,10 same as above) |
| 11 | 11 трекер |